### ISSUE TITLE
Export

### ISSUE THREAD

**Issue by maximkaneunivaka** [LendoK/Blender_GTA_V_model_importer#10]
great work and will be export plugin?

**Comment by c0nFii** [comment]
Rly Great work Man. But an Export Plugin would be very important

**Comment by DarkiBoi** [comment]
Yeah export would be really important

**Comment by Calvin-rgb** [comment]
This is a good idea. I gave up on using 3ds max.

**Comment by BrynnaDaRosa** [comment]
I cant even get access to 3DS Max 2017, the last version that can even use any of the plugins. Export would be a lifesaver.

**Comment by Axelaredz** [comment]
Hi @LendoK , 600$ we will invest in the development of an export system 😌
https://t.me/Axelaredz

**Comment by Uncle3131** [comment]
@LendoK I'm willing to make my money to make an export, our team really wants to do their animations for the game.

**Comment by BrynnaDaRosa** [comment]
I don't think you guys understand. This is just for .odd/.odr import (so far)— and we want an export for that. so far animation import is out of the question, let alone export, as much as it would be appreciated.

**Comment by LendoK** [comment]
This issue is just about exporting the mesh and skeleton files. There is an other one for animations #6 . Currently I don't have much time to work on this. Any help would be appreciated.

**Comment by Uncle3131** [comment]
> Эта проблема касается только экспорта файлов сетки и скелета. Есть и другой вариант для анимации [№6](https://github.com/LendoK/Blender_GTA_V_model_importer/issues/6) . В настоящее время у меня не так много времени, чтобы работать над этим. Любая помощь будет оценена по достоинству.

Our team understands this, but still we can't help you physically, we want to pay you to do it to the end, please.

**Comment by Uncle3131** [comment]
@LendoK Why don't you take the money and finish your business? We really want to do our animations for weapons, reloading, rack, etc.

**Comment by LendoK** [comment]
@Uncle3131 Thanks for the offer. But this is not a matter of money. This is not my business, this is a hobby project. I will finish it for free, when I find the time. I'm not even sure if I can get animations to work, sorry.

**Comment by BrynnaDaRosa** [comment]
@LendoK You should recruit for help on the gtaforums. I'm sure somebody there familiar with GIMS Evo would be helpful in the creation of export.

Meanwhile, against all odds, I got my hands on 3DS Max 2017 and have no idea how it works, so right now my mod looks like this: https://youtu.be/BmWXDghQp5I

**Comment by Dreaming-Codes** [comment]
> @LendoK Dovresti reclutare per ricevere aiuto sui gtaforum. Sono sicuro che qualcuno che abbia familiarità con GIMS Evo sarebbe utile nella creazione dell'esportazione.
> 
> Nel frattempo, contro ogni previsione, ho messo le mani su 3DS Max 2017 e non ho idea di come funzioni, quindi in questo momento la mia mod ha questo aspetto: https://youtu.be/BmWXDghQp5I

How did you export?

**Comment by BrynnaDaRosa** [comment]
> How did you export?

Not using this plugin. Not using blender. I had to export to .fbx and then import it into 3DS Max to try to export it with GIMS Evo, i have no idea what i'm doing there so it's without its materials and all sharp like that. I only have 3DS Max for the next 20 days. After that, I don't know.

**Comment by BrynnaDaRosa** [comment]
Do we know anyone that could possibly help?

**Comment by CFox04** [comment]
I've started working on exporting, and have gotten to the point where I can get simple meshes exported with no textures. Unfortunately, I am in a similar position as LendoK and do not have much time to work on this. My goal is to create a Gims Evo clone for Blender with the exact same workflow. I'm not making any promises that I will be able to complete this, but if anyone wants to help I'm open to that. I'll upload my work so far to GitHub when I get a chance.

**Comment by prateekmedia** [comment]
@CFox04 Can you upload your work on github so that other can work on it too?

**Comment by CFox04** [comment]
Sorry to not post this sooner, but I stopped the project soon after because I found out about this: https://github.com/Skylumz/Sollumz which already had a lot of progress done. This is our best hope for a blender plugin right now and already has some features you might need. They are going through a refactor right now though.
